Readme.md

```diff
@@ -75,6 +75,47 @@ install process be as simple as the `brew` process on macOS. See
 [#433](https://github.com/microsoft/scalar/issues/433) for progress on
 this feature._
 
+Installing on Linux
+-------------------
+
+Currently, we package a custom version of Git and Scalar as `.deb` packages
+that can be installed after downloading from the GitHub releases pages. The
+latest releases can be downloaded and installed as follows:
+
+```sh
+# Install microsoft/git
+wget https://github.com/microsoft/git/releases/download/v2.29.0.vfs.0.0/git-vfs_2.29.0.vfs.0.0.deb
+sudo dpkg -i git-vfs_2.29.0.vfs.0.0.deb
+
+# Install GCM Core
+wget https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v2.0.252-beta/gcmcore-linux_amd64.2.0.252.766.deb
+sudo dpkg -i gcmcore-linux_amd64.2.0.252.766.deb
```

Comment:
Note that GCM Core's package doesn't do the post install `git-credential-manager-core configure` yet, so this will be needed until it's in the package steps.